### PR TITLE
Accept image files with capitalized JPG extention in bags

### LINKED ISSUE #3243: Archival Media Bag Ingest didn't migrate JPGs for C1557
@kellybolding suspects it's because the extension is '.JPG' instead of '.jpg'.

### PR DESCRIPTION
**Pull request by hackmastera**
fixes #3243

### CODE REVIEW

File: app/models/archival_media_bag_parser.rb [hunk 1]
```diff
@@ -81,7 +81,7 @@ def pbcore_parsers
     def image_files
       @image_files ||=
         begin
-          path.join("data").each_child.select { |file| [".jpg"].include? file.extname }.map { |file| ImageFile.new(path: file) }
+          path.join("data").each_child.select { |file| [".jpg", ".JPG"].include? file.extname }.map { |file| ImageFile.new(path: file) }
```
Comment (escowles) [inline app/models/archival_media_bag_parser.rb]: Any reason not to `downcase` the extension, so any mixture of capitalization would work?

Comment (hackmastera) [reply app/models/archival_media_bag_parser.rb]: sure I can do it that way!

**Comment by tpendragon** [comment]
@hackmastera Should we have some release notes for this?

**Comment by hackmastera** [comment]
yes i'm going to add release notes! Still in draft. Ugh, the draft UI is really insufficient.

**Comment by hackmastera** [comment]
tested on staging ✅

**Pull request merged**